playbooks/robusta_playbooks/job_restart_on_oomkilled.py

```diff
@@ -9,18 +30,18 @@ class IncreaseResources(ActionParams):
     :var max_resource: This variable prevent an infinite loop of job's pod crashing and getting more memory.The action won't increase the memory again when the "Max" limit reached.
     """
 
-    increase_by: Optional[float] = 1
+    increase_by: Optional[str] = 1
```

Comment:
is "1" still a relevant value? Should it be null if optional?
Also, can you give an example in the increase_by documentation? "Users will specify how much they want to increase in each restart, e.g 500Mib"

Reply:
changed to 0.